### PR TITLE
Update README.md and add CONTRIBUTING.md for OSS release

### PR DESCRIPTION
**Pull request by jakecraige**
[Rendered Readme](https://github.com/thoughtbot/Tropos/blob/jc-oss/README.md)
[Rendered Contributing](https://github.com/thoughtbot/Tropos/blob/jc-oss/CONTRIBUTING.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+We love pull requests from everyone. Follow the thoughtbot [code of conduct]
+while contributing.
+
+[code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
+
+## Dependencies
+
+Tropos uses [Carthage][] for dependency management. Once you have it installed,
```
Comment (gfontenot) [inline CONTRIBUTING.md]: Tropos is using CocoaPods.

**Comment by gfontenot** [comment]
One fix, but yes, this is a good change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jakecraige) [inline CONTRIBUTING.md]: whoops. That's what I get for copying over files.

**Comment by gfontenot** [comment]
Looks great :+1:

**Pull request merged**